.eslintrc.json

```diff
@@ -26,7 +27,7 @@
     "react/prop-types": "off",
 
     // Prefer non-default exports
-    "import/no-default-export": "error",
+    "import/no-default-export": "off",
```

Comment:
It's to avoid errors. I was getting "Definition for rule 'import/no-default-export' was not found.", did some googling and found nothing helpful, so I turned it off. Any idea how else to handle this?

Reply:
It was messing with the handlers of the backend:
```
packages/backend/src/handlers/actions/idxCache/updateExpiredProfiles/handler.ts
/packages/backend/src/handlers/actions/idxCache/updateSingle.ts
/packages/backend/src/handlers/actions/idxCache/updateSingleProfile/handler.ts
/packages/backend/src/handlers/actions/storage/handler.ts
```
-> `Prefer named exports  import/no-default-export`